site_root/input/resources/terminology/CodeSystem-snomed-ct.json

```diff
@@ -0,0 +1,23 @@
+{

```

Comment:
Could you fix the indentation? For the JSON resources, we are using 2-space indentation. This may be useful: https://jsonformatter.org/